docs/src/modules/components/AppNavDrawerItem.js

```diff
@@ -157,7 +157,7 @@ const ItemButton = styled(Item, {
 const StyledLi = styled('li', { shouldForwardProp: (prop) => prop !== 'depth' })(
   ({ theme, depth }) => {
     return {
-      padding: depth === 0 ? '0 10px' : '4px 0',
+      padding: depth === 0 ? '0 10px' : '0',
```

Comment:
```suggestion
      padding: depth === 0 ? '0 10px' : '2px 0',
```

Removing all the padding makes it a bit harder for me to scan through it all, though. It seems better to read with a slight space between chuncks. Maybe 2px, in the middle there, it's nice.

Reply:
Okay, let's agree to disagree and compromise. 🙂